CreateSurfaceWindowsImage.ps1

```diff
@@ -3354,7 +3354,7 @@ Function Update-Win10WIM
                     }
                     Else
                     {
-                        $NewUSBDriveLetter = New-Partition -DiskNumber $TempUSB -UseMaximumSize -AssignDriveLetter | Format-Volume -FileSystem FAT32 -NewFileSystemLabel $Device
+                        $NewUSBDriveLetter = New-Partition -DiskNumber $TempUSB -Size 14GB -AssignDriveLetter | Format-Volume -FileSystem FAT32 -NewFileSystemLabel $Device

```

Comment:
Where did you get 14GB from? Why does it fix the issue?

Reply:
using -UseMaximumSize is not supported and throws error sizeNotSupported. The size should be between 14GB to 30GB to enter this block, so took the 14GB as it's the minimum size required and created the image successfully!  please let me know if you have a better solution..TIA

Reply:
Gotcha. Do you know how big the contents were after successful image creation? I just want to make sure it's not too close to 14GB, as image size will always increase with time and fluctuate a little.

Reply:
Good point, let me double check that!